spec/ataru/fixtures/db/browser_test_db.clj

```diff
@@ -138,9 +142,29 @@
   (form-store/create-new-form! form1 (:key form1))
   (form-store/create-new-form! form2 (:key form2))
   (form-store/create-new-form! form3 (:key form3))
-  (form-store/create-new-form! form3 (:key form4))
+  (form-store/create-new-form! form3 "41101b4f-1762-49af-9db0-e3603adae3ae")
   (jdbc/with-db-transaction [conn {:datasource (db/get-datasource :db)}]
     (application-store/add-application application1)
     (application-store/add-application application2)
     (application-store/add-application application3)))
 
+(defn reset-test-db [insert-initial-fixtures?]
+  (db/clear-db! :db (-> config :db :schema))
+  (migrations/migrate)
+  (when insert-initial-fixtures? (init-db-fixture)))
+
+(defn- insert-form-and-return [form]
+  (form-store/create-new-form! form (:key form))
```

Comment:
Does this not return the inserted form by default (using the `<!` suffix in the yesql function name)?